x/market/client/cli/tx.go

```diff
@@ -155,7 +156,7 @@ func cmdLeaseCreate(key string) *cobra.Command {
 				return err
 			}
 
-			id, err := BidIDFromFlagsWithoutCtx(cmd.Flags())
```

Comment:
I think we can get rid of these `*With(out)?Ctx` functions :confetti_ball:

Reply:
💯